sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -4136,6 +4136,58 @@ class Dataset[T] private[sql](
     new MergeIntoWriter[T](table, this, condition)
   }
 
+  /**
+   * Update rows in a target table that match a condition.
+   *
+   * Scala Example:
+   * {{{
+   *   spark.table("source")
+         .update(
```

Comment:
missed a `*`?

Reply:
Sorry what do you mean?  (the method also allows no condition)

Reply:
I mean we should start with a `*` in each line of the doc to ensure proper alignment and correct rendering of the documentation. You missed a `*` in the beginning of line 4145 and 4146.

Reply:
Oh sorry, i get it